generator/src/main/java/com/linkedin/pegasus/generator/FileFormatDataSchemaParser.java

```diff
@@ -60,6 +62,15 @@ public FileFormatDataSchemaParser(String resolverPath, DataSchemaResolver schema
     _schemaParserFactory = schemaParserFactory;
   }
 
+  private boolean isExtensionEntry(JarEntry entry)
```

Comment:
nit: private methods should be after public methods, also it might be useful to add a java doc to explain the logic here.

Reply:
moved